luigi/interface.py

```diff
@@ -79,7 +80,7 @@ class EnvironmentParamsContainer(task.Task):
         is_global=True, default=False,
         description='Ignore if similar process is already running')
     lock_pid_dir = parameter.Parameter(
-        is_global=True, default='/var/tmp/luigi',
+        is_global=True, default=os.path.join(tempfile.gettempdir(), 'luigi'),
```

Comment:
Indeed, it will choose `/tmp` and only than will try `var/tmp` if `/tmp` is not available. Personally I would expect people to explicitly specify `lock_pid_dir` the configuration if they rely on its location. But sure, behavior changes.  Up to you.